packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -272,6 +272,14 @@ export interface IContextualMenuListProps {
   totalItemCount: number;
   hasCheckmarks: boolean;
   hasIcons: boolean;
+  defaultMenuItemRenderer: (
+    item: IContextualMenuItem,
```

Comment:
Looking at all these arguments in here, I think that I'm in favor of moving this into a single prop object. Maybe extending `IContextualMenuItem`? It could look something like

```
interface IInternalContextualMenuItem extends IContextualMenuItem {
    index: number;
    focusableElementIndex: number;
    totalItemCount: number;
    hasCheckmarks: boolean;
    hasIcons: boolean}
```

Or something similar to that. That might help us keep props and arguments more consistent between the various render methods. Porting that to all the functions could happen later though.